.clj-kondo/config.edn

```diff
@@ -1 +1,5 @@
-{:config-paths ["../resources/clj-kondo.exports/clj-commons/aleph"]}
+{:config-paths ["../resources/clj-kondo.exports/clj-commons/aleph"]
+ :linters      {:unsorted-required-namespaces {:level :warning}}
+ :lint-as      {dynamic-redef.core/with-dynamic-redefs clojure.core/with-redefs
+                manifold.deferred/let-flow clojure.core/let
+                manifold.deferred/loop clojure.core/let}}
```

Comment:
Minor: this is fine for now, but there's a way to export kondo linting notes as part of a lib, so we should really add that to manifold instead. That way everything using Manifold will get the correct linting.

Reply:
Yep, already on my radar: https://github.com/clj-commons/aleph/pull/583#discussion_r830513450 :+1: